.github/CODEOWNERS

```diff
@@ -1 +1,5 @@
-* dsinghvi
\ No newline at end of file
+* @dsinghvi
+/generators/cli/* @armandobelardo
+/generators/commons/* @armandobelardo
+/generators/docker/* @armandobelardo
```

Comment:
i wonder if we should leave these in packages -- i like the idea of clicking on generators and just seeing recoganizable folders: postman, python, openapi, ruby, etc.

Reply:
yeah I can do that! pls hold